src/app-layout/__tests__/common.test.tsx

```diff
@@ -43,7 +43,8 @@ describeEachAppLayout({ themes: ['classic', 'refresh'] }, ({ size }) => {
       openProp: 'navigationOpen',
       hideProp: 'navigationHide',
       handler: 'onNavigationChange',
-      expectedCallsOnMobileToggle: 2,
+      // TODO: Figure out why this needs to be called twice on mobile
```

Comment:
Checked the test where this change was introduced: https://github.com/cloudscape-design/components/commit/9cd41aa0deab4c0759c613da1dbe8c9791ce2804

In general, twice events are not expected. We used to receive such bug reports before: https://github.com/cloudscape-design/components/pull/928

Good thing we fixed it on the toolbar now